README.md

```diff
@@ -0,0 +1,67 @@
+# Codic Education Server
+
+This is the code for Codic Educations server
+
+## Programs required
+MongDB - https://www.mongodb.com/try/download/community?tck=docs_server
+
+Insomnia - https://insomnia.rest/download
+
+
+## Setting Up
+
+Insomnia: 
+
+1. Open insomnia
+
+2. Go to settings -> Data -> Choose Import Data from file
+
+3. Go to the application folder -> configurations -> Insomnia 
+
+4. Double click on the newest JSON file
+
+
+MongoDB Compass:
+
+1. Go to project directory
+
+2. Create new file -> name it '.env'
+
+3. insert the following code:
+
+    DEV_DATABASE_URL = mongodb://localhost/codic
+    PROD_DATABASE_URL = TBD
+    PORT = 3001
+    ENVIROMENT = DEVELOPMENT
+
+4. Save
+
+5. Open MongoDB Compass
+
+6. Under New Connection -> click on the 'Connect' button
+
+7. Now you should be able to see the Codic database in your menu
```

Comment:
ta bort Codic. räcker med:
Now you should be able to see the database in your menu